modules/avatar/identicon/block.go

```diff
@@ -1,25 +1,24 @@
-// SPDX-License-Identifier: MIT
+// Copyright 2021 The Gitea Authors. All rights reserved.
```

Comment:
Does identicon's license allow use to do that?

Reply:
Well, that's a question. But I think MIT is not that strict. Otherwise, should we bypass the go lint?

Reply:
https://snyk.io/learn/what-is-mit-license/


The MIT license gives users express permission to reuse code for any purpose, sometimes even if code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs.

It is one of the most simple open source license agreements. The intent was for the text to be understandable by average users and to avoid extensive litigation


I think it should be fine. And we also use MIT license, and I kept the author's information here.